README.md

```diff
@@ -2,11 +2,17 @@
 ![GitHub Actions CI Status](https://github.com/nyanp/nyaggle/workflows/Python%20package/badge.svg)
 ![Python Versions](https://img.shields.io/pypi/pyversions/nyaggle.svg?logo=python&logoColor=white)
```

Comment:
How about adding a status badge of readthedocs here?

```
[![Documentation Status](https://readthedocs.org/projects/nyaggle/badge/?version=latest)](https://nyaggle.readthedocs.io/en/latest/?badge=latest)
```

[![Documentation Status](https://readthedocs.org/projects/nyaggle/badge/?version=latest)](https://nyaggle.readthedocs.io/en/latest/?badge=latest)

Reply:
Thanks! Updated.

Reply:
awesome!